docs/source/algorithms/Fit-v1.rst

```diff
@@ -178,6 +178,37 @@ Fitting to data in a MatrixWorkspace
 The error values in the input workspace are used to weight the data in
 the fit. Zero error values are not allowed and are replaced with ones.
 
+Fitting to histogram data
+#########################
+
+If the input MatrixWorkspace contains histogram data then there are two
+way to evalue the fit function on it and it can be set via the `EvaluationType`
+property. If `EvaluationType` is "CentrePoint" (which is the default) the
```

Comment:
Histogram fit only works for the Lorentzian and Gaussian, other peaks would require some sort of numerical integration. This could be an overkill for histograms with small bins and in cases when a high accuracy isn't required.